dataverse-webapp/src/main/webapp/dataverse-textMessages.xhtml

```diff
@@ -147,7 +147,7 @@
                                     <p:commandButton value="#{bundle['dataversemessages.textmessages.deactivate']}"
                                                      style="margin-right:10px;"
                                                      rendered="#{banner.active}"
-                                                     action="#{BannerTab.deactivate(banner.id)}"
+                                                     action="#{BannerTab.deactivateTextMessage(banner.id)}"
```

Comment:
IMO we should rename method `deactivateTextMessage` to `deactivateBanner` (by textMessage we mean something different)